portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path

```diff
@@ -62,7 +62,7 @@
 </tr>
 <tr>
 	<td>DIALOG_BOX</td>
-	<td>//div[contains(@class,'modal-header')]/h3</td>
+	<td>//div[contains(@class,'modal-title')]</td>
```

Comment:
@victorhcunha merged in https://github.com/liferay-bpm-qa/liferay-portal/pull/826. Can we remove this commit? Thanks!

Reply:
@patriciajeanperez commit removed!